### ISSUE TITLE
ErrorJson throws IndexOutOfRangeException

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Benknightdark/elmah#279]
```
What steps will reproduce the problem?
1. Clicking the JSON link throws an exception

What is the expected output? What do you see instead?
JSON output

What version of the product are you using? On what operating system?
Current revision 61.

Please provide any additional information below.
Fixed locally by changing Line 167 from Enumerable.Range(1, collection.Count) 
to Enumerable.Range(0, collection.Count-1)
```

Original issue reported on code.google.com by `MrStimp...@gmail.com` on 27 Feb 2012 at 4:53

**Comment by GoogleCodeExporter** [comment]
Original comment by `azizatif` on 27 Feb 2012 at 8:19
- Changed state: **Accepted**
- Added labels: **Component-Logic**, **Milestone-Release2.0**

**Comment by GoogleCodeExporter** [comment]
```
fyi I was new to Enumerable.Range() and mistook the 2nd param as "end" rather 
than count. Should be Enumerable.Range(0, collection.Count) I believe.
```

Original comment by `MrStimp...@gmail.com` on 27 Feb 2012 at 8:21

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 5775dba21e87.
```

Original comment by `azizatif` on 27 Feb 2012 at 8:21
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
No worries. I realized that and didn't take that part of the fix. ;) Thanks for 
reporting.
```

Original comment by `azizatif` on 27 Feb 2012 at 8:23
